### PR TITLE
Move command-content to js files (#779)

### LINKED ISSUE #779: Move command-content to js files
There is a function called `command-content`, and the purpose of this function is to return a nice one-line preview for commands in chat list:

screen shot 2017-01-31 at 23 34 40

Here you can see `Transaction: 0.51234 ETH`, and this text comes from `status.i18n` namespace with the help of ` status-im.chats-list.views.inner-item/command-content` function. But this text should be placed in the same place where we store all information about commands — in JS files like `commands.js` and `console.js`

### PR DESCRIPTION
**Pull request by alwx**
This PR introduces `shortPreview` JS function (it is available for DApps) and removes command-content function.

Fixes #779

### CODE REVIEW

File: src/status_im/chat/handlers/commands.cljs [hunk 1]
```diff
@@ -269,18 +269,19 @@
...
-                                         (cu/generate-hiccup result))]))
-                          (when on-requested (on-requested %)))]
+               :keys [message-id chat-id on-requested] :as message} data-type]]
+      (let [data-type (or data-type :preview)
+            path      [(if (= :response (keyword type)) :responses :commands)
+                       (if content-command content-command command)
+                       data-type]
+            params    {:parameters params
+                       :context    (merge {:platform platform/platform} i18n/delimeters)}
+            callback  #(do (log/debug "ALWX RES" data-type %) (when-let [result (get-in % [:result :returned])]
```
Comment (rasom) [inline src/status_im/chat/handlers/commands.cljs]: `(log/debug "ALWX RES"`

**Comment by alwx** [comment]
Fixes #833

File: src/status_im/translations/af.cljs [hunk 2]
```diff
@@ -148,10 +148,6 @@
    :chat-send-eth                         "{{amount}} ETH"
    :chat-send-eth-to                      "{{amount}} ETH aan {{chat-name}}"
    :chat-send-eth-from                    "{{amount}} ETH van {{chat-name}}"
-   :command-text-location                 "Ligging {{address}}"
```
Comment (rasom) [inline src/status_im/translations/af.cljs]: Where are these translations moved?

Comment (alwx) [inline src/status_im/translations/af.cljs]: All these texts are inside related js files.

**Comment by alwx** [comment]
Fixed.

@rasom

File: src/status_im/translations/af.cljs (hunk 2 above)
Comment (rasom) [inline src/status_im/translations/af.cljs]: Hm, I can find  `location_title` in js file, but I didn't find any analog of `command-text-send`, or we don't need this translation?

Comment (rasom) [inline src/status_im/translations/af.cljs]: Oh, ok, i suppose it is `send_title`

**Pull request merged**